tests/Module/CheckoutTest.php

```diff
@@ -25,12 +25,62 @@
 use Packetery\Module\RateCalculator;
 use Packetery\Module\WidgetOptionsBuilder;
 use Packetery\Nette\Http\Request;
+use PHPUnit\Framework\MockObject\Exception;
 use PHPUnit\Framework\MockObject\MockObject;
 use PHPUnit\Framework\TestCase;
 use Tests\Core\DummyFactory;
 
 class CheckoutTest extends TestCase {
 
+	private WpAdapter|MockObject $wpAdapter;
+	private WpAdapter|MockObject $wcAdapter;
+	private WpAdapter|MockObject $productEntityFactory;
+	private WpAdapter|MockObject $productCategoryEntityFactory;
+	private WpAdapter|MockObject $carrierOptionsFactory;
+	private WpAdapter|MockObject $currencySwitcherFacade;
+	private WpAdapter|MockObject $carrierEntityRepository;
+	private WpAdapter|MockObject $carDeliveryConfig;
+	private WpAdapter|MockObject $provider;
+
+	private Checkout $checkout;
+
+	public function setUp(): void {
```

Comment:
Tohle bude muset pryč. Předtím použitý způsob je preferovaný, aby se mezi jednotlivými testy nepřenášel stav.

Reply:
Ale stav se nepřenáší mezi jednotlivými testy :) vždy se ten setUp zavolá znova - tedy se ti vytvoří nová instance te classy checkout, takže nevidím důvod kor u této třídy, která má tolik závislostí to tam pytlíkovat pro každý test.

viz 3 odstavec https://docs.phpunit.de/en/10.5/fixtures.html#fixtures

Reply:
https://packeta.atlassian.net/wiki/spaces/PD/pages/413794363/General+developers+best-practices#Avoid-private-class-fields-and-setUp-methods-in-tests 

please move to a private method.